setup.cfg

```diff
@@ -98,40 +99,39 @@ all =
   %(visualization)s
 tests =
   %(all)s
-  hvpy>=1.0.1
-  hypothesis>=6.0.0  # Included in pytest-astropy. 6.0 is the first version to support disabling function-scoped fixture warning
+  hvpy
```

Comment:
What's the reason for getting rid of the pin?

Reply:
I can add it back, for me the tests should really work on two stages: latest versions of packages and the oldest deps env we create.

Reply:
Added them back